e2e/tests/core/02-client/client_test.go

```diff
@@ -133,9 +194,11 @@ func (s *ClientTestSuite) TestClientUpdateProposal_Succeeds() {
 		})
 	})
 
-	t.Run("pass client update proposal", func(t *testing.T) {
-		proposal := clienttypes.NewClientUpdateProposal(ibctesting.Title, ibctesting.Description, subjectClientID, substituteClientID)
-		s.ExecuteGovV1Beta1Proposal(ctx, chainA, chainAWallet, proposal)
+	t.Run("execute proposal for MsgRecoverClient", func(t *testing.T) {
+		authority, err := s.QueryModuleAccountAddress(ctx, govtypes.ModuleName, chainA)
```

Comment:
in the other usage of this function we do a `Require().NotNil` we should probably do the same here.